rulebook.tex

```diff
@@ -1552,21 +1582,38 @@ \subsubsection{MPS --- During Production Phase}
   \label{tab:machine-lights}
 \end{table}
 
-\subsubsection{Broken Machine Downtime}
+\subsubsection{Broken Machines}
 \label{sec:broken-machine}
 If a machine is improperly instructed or used, or a reset message has
 been sent, the machine will go into a failure state. The machine cannot be
-used for 30 seconds and until repaired. This includes interactions with shelves,
-conveyor belts, or slides.
-That is, if damage was
-inflicted on the machine or the referee needs longer to repair the
+used for 30 seconds and until repaired. This includes interactions with
+shelves, conveyor belts, or slides.
+
+If the machine reaches a failure state, the referee is prompted to confirm
+that this was caused by improper operation by the team after the timeout period
+on the refbox web interface.
+
+If the machine was used improperly, any production that was running will be
+aborted and
+any product which was being processed or placed into the machine's input or
+output is no longer available and will be removed by the referee. Any additional
+bases or caps buffered at the machine will be void and the points gained
+removed. For storage stations, no slot is refilled, the load status remains
+exactly as before the broken state. The downtime is indicated by a flashing red
+light. The machine will go into a failure state for 30 seconds.
+Should the inflicted damagetake longer to repair the
```

Comment:
the 30 seconds are redundant here.
Also I do not get the part about inflicted damage as this paragraph is talking about improper usage.
If you imply physical damage, I think this is kinda separate from this and not directly related to state handling from machines.
IMO this rather belongs in the same parts of the rulebook that talk about penalties such as pushing machines away  (basically, every alteration or damage you cause to your own production machine should not be dealt with in-game IMO, physical damage probably also constitutes a disqualification of the causing robot I presume)

Reply:
inflicted damage was originally part of this section, so I just kept it. If you think we should remove it then that's okay. However, it doesn't make sense to talk about how damage on machines is resolved in the penalties section.

Reply:
actually, you are right and I see the paragraph was also part of the old broken state. So basically in case of physical damage the referees can easily just invoke the machine to break and it should be fine.

Reply:
we never really specify consequences of damage to machines explicitly I think (given noone managed to actually do this yet I guess it is fine, but it is kind of a hole atm). There is one sentence mentioning it in the context of referee complaints and aborting games:
"The procedure
is also automatically invoked if a referee decides to abort a game for any reason (e.g., field damage,
lighting failures, burning robots)."